docs/dev_guide/recommended_development_workflow.md

```diff
@@ -22,8 +22,8 @@ The GitHub issue should contain the detailed problem statement.
 4. Run all tests as described in the project's [Workspace setup guide](../dev_guide/workspace_setup.md#run-the-test-suite).
 5. Open a pull request against the `main` branch of `linkedin/venice`. (Only in special cases would the PR be opened against other branches.)
 6. The PR title should usually be of the form `[component1]...[componentN]: Concise commit message`.
-   * Valid component tags are: `[da-vinci]`, `[server]`, `[controller]`,
-      `[router]`, `[samza]`, `[vpj]`, `[fast-client]`, `[thin-client]`, `[alpini]`,
+   * Valid tags are: `[da-vinci]` (or `[dvc]`), `[server]`, `[controller]`, `[router]`, `[samza]`,
+      `[vpj]`, `[fast-client]` (or `[fc]`), `[thin-client]` (or `[tc]`), `[changelog]` (or `[cc]`),
```

Comment:
Isn't `changelog` a feature, instead of a component?

Reply:
there will be changelog consumer client if I understand it correctly.

Reply:
"changelog consumer" will be a standalone client (like thin-client) with it's own factory, etc